src/main/java/picard/sam/SetNmMdAndUqTags.java

```diff
@@ -96,21 +96,21 @@ protected int doWork() {
 
         final ReferenceSequenceFileWalker refSeq = new ReferenceSequenceFileWalker(REFERENCE_SEQUENCE);
 
-        StreamSupport.stream(reader.spliterator(),false)
-                .peek(rec->{if(!rec.getReadUnmappedFlag()) AbstractAlignmentMerger.fixNmMdAndUq(rec, refSeq, IS_BISULFITE_SEQUENCE);})
+        StreamSupport.stream(reader.spliterator(), false)
+                .peek(rec -> {if (!rec.getReadUnmappedFlag()) AbstractAlignmentMerger.fixNmMdAndUq(rec, refSeq, IS_BISULFITE_SEQUENCE);})
```

Comment:
Everything in `SetNmMdAndUqTags.doWork()` and `CalculateUqTag.doWork()` could be refactored to a single method that accepts a `AbstractAlignmentMerger.fixNmMdAndUq()` or `AbstractAlignmentMerger.calculateUq()` method since they both have the same signature.